native/acter/src/api/spaces.rs

```diff
@@ -297,60 +310,11 @@ impl Space {
                         .await;
                 }
             },
-        );
-        self.news_entry_update_event_handle = Some(handle);
+        ));
     }
 
     pub(crate) fn remove_handlers(&mut self) {
-        let client = self.room.client();
```

Comment:
I am afraid it may be issue to not call `client.remove_event_handler()` at cleanup.